locopy/logger.py

```diff
@@ -17,46 +17,8 @@
 """Logging Module
 Module which setsup the basic logging infrustrcuture for the application
 """
-import logging
 import sys
+from loguru import logger
 
-# logger formating
-BRIEF_FORMAT = "%(levelname)s %(asctime)s - %(name)s: %(message)s"
-VERBOSE_FORMAT = (
-    "%(levelname)s|%(asctime)s|%(name)s|%(filename)s|" "%(funcName)s|%(lineno)d: %(message)s"
-)
-FORMAT_TO_USE = VERBOSE_FORMAT
-
-# logger levels
-DEBUG = logging.DEBUG
-INFO = logging.INFO
-WARN = logging.WARN
-ERROR = logging.ERROR
-CRITICAL = logging.CRITICAL
-
-
-def get_logger(name=None, log_level=logging.DEBUG):
-    """Sets the basic logging features for the application
-
-    Parameters
-    ----------
-    name : str, optional
-        The name of the logger. Defaults to ``None``
-
-    log_level : int, optional
-        The logging level. Defaults to ``logging.INFO``
-
-    Returns
-    -------
-    logging.Logger
-        Returns a Logger obejct which is set with the passed in paramters.
-        Please see the following for more details:
-        https://docs.python.org/2/library/logging.html
-    """
-    logging.basicConfig(format=FORMAT_TO_USE, stream=sys.stdout, level=log_level)
-    logger = logging.getLogger(name)
-    return logger
-
-
-# set the logger for our current dependencies
-logging.getLogger("botocore").setLevel(ERROR)
+logger.remove()
+logger.add(sys.stderr, level="INFO")
```

Comment:
Could this mess with other loggers that are already set up in a larger project?  That's one thing I'd worry about with using loguru, but I'm not very familiar with how it's supposed to work.  Their docs seem to say you should never call `add()` in a library: https://loguru.readthedocs.io/en/stable/overview.html#suitable-for-scripts-and-libraries

What I've heard for library logging is to not do the actual logger configuration for people, and leave that up to the user of the library - thoughts?

Reply:
Valid question. I was looking through some other packages which use `loguru`, like this: 
- https://github.com/williamfzc/fitch/blob/master/fitch/logger.py#L45
- https://github.com/paterit/version-checker/blob/master/updater/components.py#L20

Currently we are logging so this wouldn't be any different. If someone is going to use `locopy` I feel like providing the information by default is valuable. I get the rationale though about let users decide, but we weren't doing that initially anyway.. 🤷‍♂️

Reply:
Fair enough.  I have pulled out logging basicConfig from other libraries I've maintained at the request of users, so might want to open a separate issue at some point.  Assuming this plays nicely in cases like this then I'm a thumbs up:

```python
import logging
import sys
logging.basicConfig(stream=sys.stdout, level=logging.INFO)
logging.info('oh hi') # This should show up
logging.debug('oh hi') # This should not show up


import locopy
logging.info('oh hi') # This should show up
logging.debug('oh hi') # This should not show up
```

If someone sets up other handlers or something you may want to validate that's not messed up with the import too.

Reply:
Just tested it. It works as expected. We are calling it `logger` vs `logging`. So I think it would be isolated in that sense. I think right now it `loguru` will block other packages from showing their logging unless we explicitly enable them: `logger.enable("botocore")`